engines/tika/src/main/java/org/alfresco/transform/tika/metadata/extractors/IPTCMetadataExtractor.java

```diff
@@ -55,6 +56,12 @@ public class IPTCMetadataExtractor extends AbstractTikaMetadataExtractorEmbeddor
     private static final Pattern YEAR_IPTC = Pattern.compile("(\\d{4}[:|-]\\d{2}[:|-]\\d{2})");
 
     private ExifToolParser parser;
+    private RuntimeExec exifRuntimeExec;
+

```

Comment:
Are you sure this constructor is being called when SpringBoot is creating @Components?

Reply:
Removed the default constructor.